Accepted-RFCs/RFC-011-news-announcements.md

```diff
@@ -0,0 +1,165 @@
+
+This team would just be a formalization of who vets news posts before going on, to make sure news is on message. The ideal people for this are journalism and communication studies nerds. We happen to already be well stocked in that regard. :)
+
+### Forward facing: emailing newsletters
+
+If we want to get to a point of having a mailing list to keep our users informed, this is the content we'd use. And by having two categories, people could opt into different types of content. However, that isn't a requirement to execute on this idea.
+
+## Consequences
+
+This allows us a space to write about upcoming changes for users to experiment with (such as if we start playing in the space of having "dev branches"), announcements for recruiting, celebrating our contributors' efforts elsewhere (like announcing them being Write the Docs speakers), and upcoming meetups (like holding writing days at Write the Docs).
+
+This would be an additional effort set for the Tech Team, to take the blog and tweak it for another subfolder. That might involve refactoring material to be shared, or not—details on the code execution part of this effort is beyond the scope of the RFC, barring noting any concerns, issues, or plans that the Tech Team might share upon reviewing this.
+
+This could also play into future endeavors as we potentially expand, such as announcing Good Docs-hosted talks (which are themselves ways of passive recruiting).
+
+And this would be some additional ongoing effort on the part of the News team, though the effort should be light, and the payoff for that effort well worth it.
+
+## Links and prior art
+
+[How Write the Docs does their newsletters](https://www.writethedocs.org/blog/archive/tag/newsletter/), which as an index page feels like it works as email-delivered newsletters first, site-formatted news second.
+
+## Open questions
+
+{This section is optional and is where you can list questions that won't be resolved by this RFC, including those raised in comments from community members.}
+
+
+## Decisions deferred
+
+* Code execution (beyond anything the Tech Team wishes included in this proposal, as notes or requirements)
+* News team composition
+* Content moderation policies
+* Future news efforts (like mailing lists)
+
+## Feedback
+
+{If you accept feedback from a community member, you will incorporate it into your RFC before it is accepted.
+If you reject feedback, note that rejected feedback here before resolving the conversation.}
+
+## Organizational dependencies
+
+* Content strategy in involved in such decisions & nuance
+* The tech team would be involved in implementation
+
+## Implementation checklist
+
+If this proposal is accepted, the following tasks must be completed:
+
+- [ ] Set up the site folder
+- [ ] Document process for posting
+- [ ] Document the subsite's purpose and content guidelines
+
```

Comment:
Define a new 'content type' that would have the necessary data structure for this kind of information object.

Reply:
I'll make a task category of "technical implementations" and add this (and move "set up folder" etc.) in that. And make "Create process" as a task category.